scripts/check-i18n-naming.js

```
@@ -0,0 +1,217 @@
+/**
+ * Main validation function.
+ * Reads the common.json file, checks for violations, and reports results.
+ */
+const runValidation = () => {
+  console.log('Checking i18n key naming conventions...\n');
+
+  try {
+    const filesToCheck = localeFilePaths.length ? localeFilePaths : [filePath];
+    const missing = filesToCheck.filter((file) => !fs.existsSync(file));
+    if (missing.length > 0) {
+      missing.forEach((file) => console.error(`❌ File not found: ${file}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)